test_purl.py

```diff
@@ -28,6 +28,10 @@
 import json
 import re
 import unittest
+import os
+src= os.path.abspath("packageurl.py")
+import sys
+sys.path.append("src")
```

Comment:
You never want to modify the sys.path in a library
Do you know about virtualenv?